recipes/libhdbpp/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "libhdbpp" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/tango-controls/hdbpp/libhdbpp/-/archive/v{{ version }}/libhdbpp-v{{ version }}.tar.gz
+  sha256: 53e1647d0106787adaed5d9e82c3eb53c5d5b1a4892387b1e52a183fadde4b93
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('cxx') }}
+    - libtool
+    - pkg-config
+  host:
+    - cppzmq
+    - cpptango
```

Comment:
The [logs shows](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=521716&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1341) that you are probably statically linking against cpptango.

Reply:
The `cpptango` package only contains a shared library and some headers.
Looks like `libhdbpp` is only creating an interface library: https://gitlab.com/tango-controls/hdbpp/libhdbpp/-/blob/master/cmake/FindTango.cmake#L15, so it's not linking against it.

`cpptango` is needed to compile this library and it defines a `run_exports`. Should I ignore this `run_exports`?
`libhdbpp` is an interface library and will be used to compile other packages.

Reply:
If there are no linkage/symbols from libhdb++.so to/in libcpptango.so, then it is not a runtime dependency of this library and the run_export should be ignored.

I'm not sure what you mean by interface library. If downstream packages need to link against cpptango, they should explicitly add it to their recipe host requirements.

Reply:
OK, thanks. `run_exports` ignored.